snippets/nix.json

```diff
@@ -0,0 +1,85 @@
+{
+    "ifelse": {
+        "prefix": "if",
+        "body": [
+            "if $1 then",
+            "  $2",
+            "else$0"
+        ],
+        "description": "if-else block"
+    },
+    "letin": {
+        "prefix": "let",
+        "body": [
+            "let",
+            "  $1",
+            "in $0"
+        ],
+        "description": "let-in block"
+    },
+    "hash": {
+        "prefix": "hash",
+        "body": "hash = \"sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=\";",
```

Comment:
Should the body be `hash = ${1:sha256-AAA...}`?

Reply:
> Should the body be `hash = ${1:sha256-AAA...}`?

Yea why not :). Thanks.

Reply:
Pushed this change.